kroma-devnet/devnet/__init__.py

```diff
@@ -163,6 +183,46 @@ def deploy_contracts(paths):
 
     shutil.copy(paths.l1_deployments_path, paths.addresses_json_path)
 
+# [Kroma: START]
+def deploy_contracts_e2e(paths):
```

Comment:
Most code blocks of this function is duplicate. How about unifying two functions and add some arguments to use in both cases?

Reply:
What I first thought was that this kind of separating would be temporary, since it makes the management of e2e and devnet harder, and certainly we should sometime remove the `ValidatorPool` and `ValPoolTerminationIndex` to reduce gas after the successful transition to Validator System V2. So it was meant to make the reverting process easier. How do you think?

Reply:
Ok I agree. Then can you add a comment to the description of `e2e_devnet_l1_genesis` function that mentions this PR and the need to be reverted of the changes in this PR after the transition?

Reply:
Yep I'll add it, thanks